.github/workflows/continuous-integration-workflow.yml

```diff
@@ -12,30 +12,27 @@ jobs:
       matrix:
         distro: ['fedora:latest', 'fedora:rawhide', 'ubuntu:latest']
         cxx: ['g++', 'clang++']
+        cxx_extra_flags: ['']
         cmake_build_type: ['Release', 'Debug']
         backend: ['OPENMP']
         clang-tidy: ['']
         include:
           - distro: 'fedora:intel'
             cxx: 'icpc'
+            cxx_extra_flags: '-diag-disable=177,10441'
```

Comment:
This avoids
```
warning #177: member "<unnamed>::defaultdevicetype_cmd_line_args_unrecognized_flag_Test::test_info_" was declared but never referenced
```
which we seemingly get for every test.

Reply:
Do these actually run?

Reply:
Yes, see https://github.com/masterleinad/kokkos/actions/runs/4000209392/jobs/6865047046.

Reply:
No I mean the actual executable output `defaultdevicetype.cmd_line_args_unrecognized_flag` from gtest

Reply:
The error was also there for `defaultdevicetype.initialization_settings` so I picked that one to just `abort()` in it.

Reply:
> Related, I think it would be good to build with `--verbose` to see if the flag are actually there.

ping on that, so I can see the flags it is build with.

Reply:
The flags passed to `CMake` are in the logs.

Reply:
See https://github.com/masterleinad/kokkos/actions/runs/4001195807/jobs/6867188598 for a run with `ctest -V` that shows `defaultdevicetype.cmd_line_args_unrecognized_flag` and `defaultdevicetype.initialization_settings` being run (among others).